src/Chainweb/Payload/RestAPI.hs

```diff
@@ -142,6 +148,25 @@ payloadPostApi
     . Proxy (PayloadPostApi v c)
 payloadPostApi = Proxy
 
+-- | @POST \/chainweb\/\<ApiVersion\>\/\<InstanceId\>\/chain\/\<ChainId\>\/heightedpayload\/batch@
+--
+-- The query may return any number (including none) of the requested payload
+-- data. Results are returned in any order.
+--
+type HeightedPayloadPostApi_
+    = "payload"
+    :> "batch"
+    :> ReqBody '[JSON] [(BlockHeight, BlockPayloadHash)]
```

Comment:
I think tuples are serialized as lists, Is that what we want, or would we rather prefer an object with meaningful property name?